gremlin-go/driver/result.go

```diff
@@ -21,18 +21,190 @@ package gremlingo
 
 // AN-968 Finish Result implementation.
 
-import "fmt"
+import (
+	"errors"
+	"fmt"
+	"reflect"
+	"strconv"
+)
 
 // Result Struct to abstract the Result and provide functions to use it.
 type Result struct {
 	result interface{}
 }
 
-// AsString returns the current Result formatted as a string.
-func (r *Result) AsString() string {
+// ToString returns the string representation of the Result struct in Go-syntax format
+func (r *Result) ToString() string {
+	return fmt.Sprintf("result{object=%v class=%T}", r.result, r.result)
+}
+
+// GetString gets the string representation of the result, equivalent to GetString()
```

Comment:
Are you certain `GetString` is equivalent to `GetString()`?
```suggestion
// GetString gets the string representation of the result
```